tests/network/test_network_actors.py

```diff
@@ -48,21 +48,34 @@ def test_vladimir_illegal_interface_key_does_not_propagate(ursulas):
     assert digest(illegal_key) in ursula.server.protocol.illegal_keys_seen
 
 
-def test_alice_finds_ursula(alice, ursulas):
+def test_alice_finds_ursula_via_dht(alice, ursulas):
     """
     With the help of any Ursula, Alice can find a specific Ursula.
     """
     ursula_index = 1
     all_ursulas = blockchain_client._ursulas_on_blockchain
     value = alice.server.get_now(all_ursulas[ursula_index])
-    header, _signature, _ursula_pubkey_sig, _hrac, interface_info = dht_value_splitter(value,
+    header, _signature, _ursula_pubkey_sig, interface_info = dht_value_splitter(value,
                                                                                return_remainder=True)
 
     assert header == constants.BYTESTRING_IS_URSULA_IFACE_INFO
-    port = msgpack.loads(interface_info)[0]
+    port = msgpack.loads(interface_info)[1]
     assert port == URSULA_PORT + ursula_index
 
 
+def test_alice_finds_ursula_via_rest(alice, ursulas):
+    networky_stuff = MockNetworkyStuff(ursulas)
```

Comment:
I'm confused why we are still calling it `networky_stuff` when it's supposed to be `network_middleware`?